RiskAssessmentApi.yaml

```
@@ -248,9 +376,195 @@ components:
       - directionDegrees
       - speedMetersPerSecond
 
+    ObstacleProximityRiskThresholds:
+      type: object
+      description: A collection of thresholds for setting obstacle proximity risk level
+      properties:
+        minimumDistanceMeters:
+          type: number
+          format: double
+          minimum: 0.0
+          description: The threshold for the minimum distance to any obstacle in meters
+        obstacleCount:
+          type: number
+          format: double
+          minimum: 0
+          description: The threshold for the number of obstacles within the minimum distance
+      required:
+      - minimumDistanceMeters
+      - buildingCount
+
+    GroundImpactRiskThresholds:
+      type: object
+      description: A collection of thresholds for setting ground impact risk level
+      properties:
+        probabilityOfCasualty:
+          type: number
+          format: double
+          minimum: 0.0
+          maximum: 1.0
+          description: The threshold for the probability of casualty for ground impact impact
+      required:
+      - probabilityOfCasualty
+
+    NavigationReliabilityRiskThresholds:
+      type: object
+      description: A collection of thresholds for setting navigation reliability risk level
+      properties:
+        satelliteCount:
+          type: integer
+          format: int32
+          minimum: 0
+          description: The threshold for the number of visible satellites
+        pdop:
+          type: number
+          format: double
+          minimum: 0.0
+          description: The threshold for the position dilution of precision (PDOP) indicating the strength of the satellite constellation for determining position
+        weightedRisk:
+          type: number
+          format: double
+          minimum: 0.0
+          maximum: 1.0
+          description: The threshold for the weighted risk which incorporates satellite count, PDOP, and map tile availability
+      required:
+      - satelliteCount
+      - positionDilutionOfPrecision
```

Either property should be 'positionDilutionOfPrecision' or this should be 'pdop'. I'm guessing the latter
```suggestion
      - pdop
```